e2e/node/unauth.test.ts

```diff
@@ -0,0 +1,287 @@
+/**
+ * Copyright 2022 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import {
+  jest,
+  describe,
+  it,
+  expect,
+} from "@jest/globals";
+
+import { foaf, schema } from "rdf-namespaces";
+import {
+  getSolidDataset,
+  setThing,
+  getThing,
+  getStringNoLocale,
+  setDatetime,
+  setStringNoLocale,
+  saveSolidDatasetAt,
+  isRawData,
+  getContentType,
+  getResourceInfoWithAcl,
+  getSolidDatasetWithAcl,
+  hasResourceAcl,
+  getPublicAccess,
+  getAgentAccess,
+  getFallbackAcl,
+  getResourceAcl,
+  getAgentResourceAccess,
+  setAgentResourceAccess,
+  saveAclFor,
+  hasFallbackAcl,
+  hasAccessibleAcl,
+  createAclFromFallbackAcl,
+  getPublicDefaultAccess,
+  getPublicResourceAccess,
+  getFile,
+  getSourceUrl,
+  deleteFile,
+  createContainerAt,
+  createContainerInContainer,
+  getBoolean,
+  setBoolean,
+} from "../../src/index";
+
+// This block of end-to-end tests should be removed once solid-client-authn-node works against NSS,
+// and the e2e tests have an NSS environment setup.
+describe("End-to-end tests with pre-existing data against resources in an NSS server", () => {
```

Comment:
Seeing what these tests intend to do, wouldn't it be more accurate to name the file `nss-unauth` or something ?

Reply:
Renamed in https://github.com/inrupt/solid-client-js/pull/1426/commits/956ff574950f6b57711d88249b8d2bcfa7a2276e.